README.md

```diff
@@ -90,6 +91,78 @@ Sure! However you will need a [SPID account](https://www.agid.gov.it/en/platform
+| com.sec.android.provider.badge.permission.READ                          | used for the notification badge                                                   |
+| com.sec.android.provider.badge.permission.WRITE                         | used for the notification badge                                                   |
+| com.sonyericsson.home.permission.BROADCAST_BADGE                        | used for the notification badge                                                   |
+| com.sonymobile.home.permission.PROVIDER_INSERT_BADGE                    | used for the notification badge                                                   |
+| me.everything.badger.permission.BADGE_COUNT_READ                        | used for the notification badge                                                   |
+| me.everything.badger.permission.BADGE_COUNT_WRITE                       | used for the notification badge                                                   |
+
+
+**ios**
+(more info about permissions requested by Apple [here](https://www.pivotaltracker.com/n/projects/2048617/stories/160232897))
+
+| Permission                                  | Usage / Meaning                                                                                                                          |
+| ------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| NSAppleMusicUsageDescription                | This key is required if your app uses APIs that access the user’s media library.                                                         |
+| NSBluetoothAlwaysUsageDescription           | This key is required if your app uses the device’s Bluetooth interface.                                                                  |
+| NSBluetoothPeripheralUsageDescription       | This key is required if your app uses APIs that access Bluetooth peripherals and has a deployment target earlier than iOS 13.            |
+| NSContactsUsageDescription                  | IO needs access to your contacts to let you add them in calendar events.                                                                 |
+| NSLocationAlwaysUsageDescription            | This key is required if your iOS app uses APIs that access the user’s location at all times and deploys to targets earlier than iOS 11.  |
+| NSLocationUsageDescription                  | This key is required if your app uses APIs that access the user’s location information.                                                  |
+| NSLocationWhenInUseUsageDescription         | This key is required if your iOS app uses APIs that access the user’s location information while the app is in use.                      |
+| NSMicrophoneUsageDescription                | IO needs access to the microphone in case you want to leave a voice note.                                                                |
```

Comment:
```suggestion
| NSMicrophoneUsageDescription                | IO needs access to the microphone in case you want to leave a voice note. Used in the assistance flow                                                               |
```